README.md

```diff
@@ -8,22 +8,54 @@ ML.NET allows .NET developers to develop their own models and infuse custom mach
 
 ML.NET was originally developed in Microsoft Research, and evolved into a significant framework over the last decade and is used across many product groups in Microsoft like Windows, Bing, PowerPoint, Excel and more.
```

Comment:
It used for both training and production, should we add that it is a production quality ML framework?

Reply:
Agree. I modified the text including that perspective. 👍